gallery/src/styles/PageLayout/SingleColumnLayoutExample.html

```diff
@@ -0,0 +1,18 @@
+<!--
+
+    Copyright (c) 2019-present Sonatype, Inc.
+    This program and the accompanying materials are made available under
+    the terms of the Eclipse Public License 2.0 which accompanies this
+    distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+
+-->
+<body class="nx-body">
+  <div class="nx-page">
+    <div class="nx-page-header">
+    </div>
+    <div class="nx-page-content">
+      <div class="nx-page-content-main">
+      </div>
+    </div>
```

Comment:
> In keeping with our recommendation that `.nx-page` be the react root, I'm wondering if we should demonstrate its children as react components instead of HTML.

If we did that then I'd suggest we have two sections, the first is just the HTML skeleton illustrated, the second is how you'd break up that skeleton into react components.

Reply:
> If we did that then I'd suggest we have two sections, the first is just the HTML skeleton illustrated, the second is how you'd break up that skeleton into react components.

Eh nevermind.  Recommending "this is where the react part should start" is one thing, but I don't really want to be recommending how apps should be breaking down their react code into different components.